### ISSUE TITLE
Sound turns off

### ISSUE THREAD

**Issue by alexvorxx** [longjunyu2/winlator#155]
Hi.
The sound stops working when you disconnect/connect headphones, as well as when there is an incoming call.
This happens with both ALSA and PulseAudio.

**Comment by alexvorxx** [comment]
Logcat shows this when headphones disconnected:
`926  2258 D audio_hw_primary: out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: appname=-com.winlator`
[Log_2024-11-12_08-50-08.txt](https://github.com/user-attachments/files/17711285/Log_2024-11-12_08-50-08.txt)

**Comment by pavankatari** [comment]
> Hi. The sound stops working in Winlator when you disconnect/connect headphones, as well as when there is an incoming call. This happens with both ALSA and PulseAudio.

Yes I'm also facing this issue from orginal winlator too, hope fix soon

**Comment by alexvorxx** [comment]
This problem is fixed in Winlator 9.0 (with ALSA only).

**Comment by alexvorxx** [comment]
Problem with ALSA may be fixed with this [commit](https://github.com/alexvorxx/winlator/commit/d243aa0a6aecb3fa705a67716dfbdc6d26bfea7f).
